### DISCUSSION TITLE
AGH login disable?

### DISCUSSION THREAD

**Discussion by bcskillings** [hassio-addons/addon-adguard-home#152] [Q&A] [answered]
I'm looking to make a similar add on for Cockpit and would like to disable the login so I do not have to login to Cockpit and then AGH. Y'all seem to have gotten that done properly but I can't seem to find that point in the code, can someone point me to the correct area?

**Comment by frenck** [comment, marked as answer]
See: <https://github.com/hassio-addons/addon-adguard-home/blob/main/adguard/DOCS.md#option-leave_front_door_open>

**Comment by bcskillings** [comment]
Perfect, thank you!
